spec/factories/users.rb

```diff
@@ -14,6 +14,7 @@
     company
     otp_required_for_login { false }
     office                { create(:office, company: company) }
+    city                  { create(:city) }
```

Comment:
you don't need to pass the block for relation

Reply:
done